### PR TITLE
Add corresponding translations in locale en & fr

### PR DESCRIPTION
**Pull request by nuhuhmutebi**
Main files edited
1. /clinical/locale_en.json
2. /clinical/locale_en.json

### CODE REVIEW

File: configuration/openmrs/i18n/clinical/locale_en.json [hunk 1]
```diff
@@ -87,6 +87,7 @@
 
   "CLINICAL_ORDER_RADIOLOGY_NEED_PRINT": "Need Print for this order.",
   "CLINICAL_ORDER_RADIOLOGY_NEED_PRINT_BUTTON": "Needs Print",
+  "CLINICAL_ACCEPT": "Validate",
```
Comment (reagan-meant) [inline configuration/openmrs/i18n/clinical/locale_en.json]: Could you change this key to [this](https://github.com/mekomsolutions/bahmni-config-haiti/search?q=CLINICAL_ACCEPT)

Comment (nuhuhmutebi) [reply configuration/openmrs/i18n/clinical/locale_en.json]: Done

File: configuration/openmrs/i18n/clinical/locale_fr.json [hunk 2]
```diff
@@ -110,6 +110,7 @@
   "CONSULTATION_TAB_NEW_DIAGNOSES_LABEL": "Récemment ajouté",
   "CONSULTATION_TAB_OBSERVATIONS_HEADER_LABEL": "Observations",
   "CONSULTATION_TAB_PAD_EMPTY_MESSAGE": "Aucune note de consultation",
+  "CONSULTATION_PAD_EMPTY_MESSAGE": "Aucune consultation Notes",
```
Comment (rbuisson) [inline configuration/openmrs/i18n/clinical/locale_fr.json]: "Aucune note de consultation"

File: configuration/openmrs/apps/clinical/medication.json [hunk 3]
```diff
@@ -0,0 +1,172 @@
+{
+  "commonConfig": {},
```
Comment (rbuisson) [inline configuration/openmrs/apps/clinical/medication.json]: What exactly differs in this file from Haiti one?
Isn't there a way to get things aligned? If not, then that's fine.

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: configuration/openmrs/apps/clinical/medication.json (hunk 3 above)
Comment (nuhuhmutebi) [reply configuration/openmrs/apps/clinical/medication.json]: Deleted `medication.json` to inherit directly from haiti

File: configuration/openmrs/i18n/clinical/locale_fr.json (hunk 2 above)
Comment (nuhuhmutebi) [reply configuration/openmrs/i18n/clinical/locale_fr.json]: Done

**Comment by reagan-meant** [comment]
No longer necessary. Fixed Haiti level configs

**Pull request closed without merging**